packages/fuels-rs/Cargo.toml

```diff
@@ -7,6 +7,7 @@ homepage = "https://fuel.network/"
 license = "Apache-2.0"
 repository = "https://github.com/FuelLabs/fuels-rs"
 description = "Fuel Rust SDK."
+rust-version = "1.58.0"
```

Comment:
Should this conform to https://github.com/rust-dev-tools/fmt-rfcs/blob/master/guide/cargo.md?

Reply:
Is there a `Cargo.toml` formatting lint we can use in CI?

Reply:
https://github.com/FuelLabs/fuel-core/blob/670d16379ed1420061fc9772043b86e7101a3a6e/.github/workflows/ci.yml#L33-L39

Reply:
Which is actually https://github.com/FuelLabs/cargo-toml-lint!

Reply:
OK, I think this has been addressed! It was just the ordering of fields right?

Reply:
Ahh, I just need to resolve conflicts